src/slipcover/__main__.py

```diff
@@ -17,117 +17,125 @@
 # showing it what we need.
 #
 import argparse
-ap = argparse.ArgumentParser(prog='slipcover')
```

Comment:
Looks like I need to wrap this big piece of code in a function, setuptools needs it to create an entrypoint.